parameterized/parameterized.py

```diff
@@ -426,8 +449,16 @@ def parameterized_expand_wrapper(f, instance=None):
             paramters = cls.input_as_callable(input)()
             for num, p in enumerate(paramters):
                 name = name_func(f, num, p)
-                frame_locals[name] = cls.param_as_standalone_func(p, f, name)
+                # If original function applied any patch, re-construct
```

Comment:
Please change this to `# If the original function has patches applied by 'mock.patch', then re-construct…`
(ie, to make it explicit that these are patches from `mock`).